pkg/trace/api/otlp_test.go

```diff
@@ -1056,9 +1056,8 @@ func TestOTLPConvertSpan(t *testing.T) {
 					"span.kind":               "server",
 				},
 				Metrics: map[string]float64{
-					"_top_level": 1,
```

Comment:
Should we add a test case that still checks that `_top_level` span is part of the metrics when `enable_otlp_compute_top_level_by_span_kind` is configured?